README.md

```diff
@@ -92,21 +94,21 @@ pip install -e '.[dev]'
 ### Optional Dependencies
 There are several dependency groups that can be selected during pip installation, like
 ```bash
-pip install baybe[test,lint] # will install baybe with additional dependency groups `test` and `lint`
+pip install 'baybe[test,lint]' # will install baybe with additional dependency groups `test` and `lint`
 ```
 To get the most out of `baybe`, we recommend to install at least
 ```bash
-pip install baybe[chem,simulation]
+pip install 'baybe[chem,simulation]'
 ```
 
 The available groups are:
-- `chem`: Cheminformatics utilities (e.g. for the [`SubstanceParameter`](baybe.parameters.substance.SubstanceParameter)).
+- `chem`: Cheminformatics utilities (e.g. for the [`SubstanceParameter`](https://emdgroup.github.io/baybe/_autosummary/baybe.parameters.substance.SubstanceParameter.html#baybe.parameters.substance.SubstanceParameter)).
 - `docs`: Required for creating the documentation.
 - `examples`: Required for running the examples/streamlit.
 - `lint`: Required for linting and formatting.
 - `mypy`: Required for static type checking.
-- `onnx`: Required for using custom surrogate models in ONNX format.
-- `simulation`: Enabling the [`simulation`](baybe.simulation) module.
+- `onnx`: Required for using custom surrogate models in [ONNX format](https://onnx.ai).
+- `simulation`: Enabling the [`simulation`](https://emdgroup.github.io/baybe/_autosummary/baybe.simulation.html) module.

```

Comment:
do you really like the link-code combinations now everywhere? imo they look bad and Id go with either link OR code, but not both

Reply:
fixed